modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt

```diff
@@ -88,7 +100,7 @@ fun String.asAssetId() = this.split(ASSET_ID_DELIMITER).takeIf {
     }
 } ?: throw IllegalArgumentException("Incorrect asset ID: $this")
 
-fun String.asTokenId() = PermissionTokenId(Name(this))
+// fun String.asTokenId() = PermissionTokenId(Name(this))
```

Comment:
I think it should either be commented or deleted

Reply:
deleted